Tools/JSObjects/CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "GNU" OR
       COMPILE_FLAGS -Wno-error=unused-but-set-variable)
   target_compile_options(jsobjects PRIVATE -Wall -Wextra -Werror)
   target_compile_options(jsobjects PRIVATE -Wno-unused-parameter
+                                           -Wno-unknown-warning-option
```

Comment:
Can we limit this to Android or under a version check? If we can keep the clang versions homogenous, it makes it better for diagnosing portability issues to have all the warnings be similar.